saleor/graphql/app/types.py

```diff
@@ -217,18 +225,18 @@ def resolve_permissions(root: models.App, _info, **_kwargs):
         return format_permissions_for_display(permissions)
 
     @staticmethod
-    @permission_required(AppPermission.MANAGE_APPS)
-    def resolve_tokens(root: models.App, _info, **_kwargs):
-        return root.tokens.all()  # type: ignore
+    def resolve_tokens(root: models.App, info, **_kwargs):
+        has_required_permission(root, info.context)
```

Comment:
I think here you could keep the previous implementation. The new one only adds OWNER but it does nothing in query resolvers, it's just a way to represent the fact of being "owner" in the permission descriptions.

Reply:
The previous resolver was requiring `MANAGE_APPS` permission and we want to allow an app to access to own resources. Can you give a bit more context how to use the previous way in that case?

Reply:
Sorry, I missed the fact that `requestor_has_access` checks the ownership, so this is OK.